spec/features/chitter_features_spec.rb

```diff
@@ -0,0 +1,56 @@
+# STRAIGHT UP
+#
+# As a Maker
+# So that I can let people know what I am doing
+# I want to post a message (peep) to chitter
+#
+feature "Posting a peep" do
+  scenario 'Users can post a new peep' do
+    visit('/post')
+    fill_in :peep, with: "Test peep, 123"
+    click_button "Peep!"
```

Comment:
Had a nasty case of false green for this.
If your button is broken for any reason, this can pass a false green because your text will still be in :peep